### PR TITLE
Castaway/html caching again

### LINKED ISSUE #998: Load message content first, then attachments and images
When opening an HTML message with attachments or images, the attachment/images are shown first and later the content.

Instead the textual content should be shown first, and then the attachments/images.

See also https://github.com/runbox/runbox7/issues/616 and https://github.com/runbox/runbox7/issues/611.

### PR DESCRIPTION
**Pull request by castaway**
* Pre-load HTML content by including it in the initial "fetch message data" request - loads HTML emails faster 
* Trust backend pre-sanitized HTML, displays in original format in HTML view
*  Should not get security warnings on Replies or Forwards of HTML emails
* Reply/Fwd HTML email displays in original format

Fixes #998 #1020

### CODE REVIEW

File: src/app/compose/compose.component.ts [hunk 1]
```diff
@@ -123,15 +123,21 @@ export class ComposeComponent implements AfterViewInit, OnDestroy, OnInit {
...
                     this.has_pasted_signature = true;
                     this.signature = from.signature;
                     this.model.msg_body = this.signature.concat('\n\n', this.model.msg_body);
+                    if ( from.is_signature_html ) {
+                        this.model.useHTML = true;
+                        this.model.html = this.signature.concat('\n\n', this.model.html || this.model.msg_body);
+                    }
                 }
             }
+            // This.. shouldnt happen as only have content if reply/fwd
```
Comment (tadzik) [inline src/app/compose/compose.component.ts:136]: DId it happen anyway in testing? What's the story here?

Comment (castaway) [reply src/app/compose/compose.component.ts:136]: Nope, it just seemed illogical to leave out that check...

Comment (tadzik) [reply src/app/compose/compose.component.ts:136]: Fair enough :)

**Review by tadzik — APPROVED** [review]

**Review by gtandersen — APPROVED** [review]
Nice work @castaway!

**Pull request merged**